resources/lang/en/general.php

```diff
@@ -41,7 +41,6 @@
     'consumables'			=> 'Consumables',
     'country'  				=> 'Country',
     'create'				=> 'Create New',
-    'created'               => 'Item Created',
```

Comment:
I think you might need to do a fresh pull? We don't want to delete these strings.